lib/packwerk/reference_checking/checkers/privacy_checker.rb

```diff
@@ -11,6 +11,39 @@ class PrivacyChecker
 
         VIOLATION_TYPE = T.let("privacy", String)
 
+        class << self
```

Comment:
Does it make sense to put this in a nested different class or module? This logic isn't really related to the checker, it belongs to the package.

Reply:
Eventually, this whole class will implement a `Packwerk::PluginInterface`, which will connect it to the rest of the packwerk runtime. I think that it might make sense to better factor out the privacy implementation into more logical parts when it comes time to do that, but I was hoping at this stage to just move all privacy related things into a pile in one place so that it can be deleted from packwerk.

Let me know if you think think that makes sense as a stepping stone.